packages/esm-ohri-pmtct-app/src/views/child-health/tabs/infant-postnatal-care.component.tsx

```diff
@@ -48,18 +53,11 @@ const InfantPostnatalList: React.FC<InfantPostnatalListProps> = ({ patientUuid }
           return getObsFromEncounter(encounter, pTrackerIdConcept);
         },
       },
-      {
-        key: 'childDateOfBirth',
-        header: t('childDateOfBirth', 'Child Date of Birth'),
-        getValue: (encounter) => {
-          return getObsFromEncounter(encounter, childDateOfBirth, true);
-        },
-      },
       {
         key: 'mothersName',
         header: t('mothersName', 'Mothers Name'),
```

Comment:
```suggestion
        header: t('mothersName', 'Mother's Name'),
```

Reply:
This would be better in the en.json translation file otherwise this doesn't work. Will change it there